teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -81,6 +81,17 @@
       arity = "1")
   private Bytes32 graffiti;
 
+  @Option(
+      names = {"--validator-performance-tracking-enabled"},
+      paramLabel = "<BOOLEAN>",
+      description = "Enable validator performance tracking and logging",
+      arity = "1")
+  private boolean validatorPerformanceTrackingEnabled = false;
```

Comment:
much better off using 
```
fallbackValue = "true",
arity = "0..1"
```
so that `--validator-performance-tracking-enabled` works to turn it on, and accepts =false to explicitly turn off. lots of examples in p2pOptions, loggingOptions...

Reply:
i'd be tempted to fix the locking-enabled while you're there...

Reply:
Done.